hms-app/src/components/HeaderMenu.tsx

```diff
@@ -227,8 +227,9 @@ export default function HeaderMenu({
     </div>
   )
 
-  const closeEditModal = (isOpened: boolean) => {
-    setEditModalOpened(isOpened)
+  const closeEditModal = () => {
```

Comment:
now that name doesnt fit anymore

Reply:
is it because it also refresh the list? So the function name doesn't fit

Reply:
yeah thats what I mean 
its not closing the edit modal as the name suggest, its more like refreshAfterChange or somethink like that